scripts/pdf/README.md

```diff
@@ -34,5 +34,7 @@ Complete information about the arguments can be viewed by running:
 The color-schemes that can be specified are:
```

Comment:
Optional fix: separate PR into 2 ones based on directories changed: everything in `scripts/pdf` can be put in a separate PR.